text.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version:  >= 1.10
+cabal-version:  2.2
 name:           text
 version:        1.2.5.0
```

Comment:
What version will this end up as? 2.0? How exciting!

Reply:
Heck yeah! We get common stanzas now :)

Reply:
Yes, some packages have bounds like `text >= 1 && < 2`, so to be on the safe side let's bump all the way to `text-2.0`.